vertical-pod-autoscaler/pkg/recommender/main.go

```diff
@@ -55,7 +55,7 @@ var (
 	kubeApiQps             = flag.Float64("kube-api-qps", 5.0, `QPS limit when making requests to Kubernetes apiserver`)
 	kubeApiBurst           = flag.Float64("kube-api-burst", 10.0, `QPS burst limit when making requests to Kubernetes apiserver`)
 
-	storage = flag.String("storage", "", `Specifies storage mode. Supported values: prometheus, checkpoint (default)`)
+	storage = flag.String("storage", "", `Specifies storage mode. Supported values: prometheus, none, checkpoint (default)`)
```

Comment:
Why default value is "" and not "checkpoint" ?

Reply:
changed

<img width="1073" alt="image" src="https://github.com/kubernetes/autoscaler/assets/16960194/166ea5e5-505a-4e78-a726-23639779a3bc">

Reply:
The default value is still "". The second argument of `flag.String()` is the default value of the flag the call defines: https://pkg.go.dev/flag#String

Reply:
that has to be github taking you to the outdated commits via permalink, please see this one
https://github.com/Evedel/autoscaler/blob/add-strict-mode-to-force-retries-during-prometheus-history-loading/vertical-pod-autoscaler/pkg/recommender/main.go#L60